mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -279,82 +379,63 @@ export function PlaceBidScreen(props: Props): JSX.Element {
   );
 }
 
-function BidSummaryCard(props: {
-  displaySymbol: string;
-  collateralDisplaySymbols: string[];
-  blockCount: number;
-  liquidationHeight: number;
-  minNextBid: string;
-  totalAuctionValue: string;
-  onPressFullDetails: () => void;
-  minNextBidInUSD: string;
-}): JSX.Element {
+enum BidAmountButtonTypes {
+  Min = "100%",
+  One = "101%",
+  Two = "102%",
+  Five = "105%",
+}
+interface PercentageAmountButtonProps {
+  onPress: (value: string, type: BidAmountButtonTypes) => void;
+  bidPercentageAmount: BidAmountButtonTypes;
+  minNextBidInToken: string;
```

Comment:
if we do this, we can remove line 400, and dont have to pass in `.toFixed()` at line 242
```suggestion
  minNextBidInToken: BigNumber;
```

Reply:
Updated, followed this suggestion. Thanks!